README.md

```diff
@@ -4,13 +4,22 @@ Arch Linux packaging for [ungoogled-chromium](//github.com/Eloston/ungoogled-chr
 
 ## Downloads
 
-Available in the AUR as `ungoogled-chromium`
+Available in the AUR as `ungoogled-chromium`, wayland users could benefit from using `ungoogled-chromium-ozone` and `ungoogled-chromium-git` is for the bleeding edge experience.
```

Comment:
Probably better to format this as a list so you can explain the differences between these packages in more detail.

Reply:
understandable, will do